src/main/webapp/app/exam/manage/student-exams/student-exam-timeline/programming-exam-diff/programming-exercise-exam-diff.component.html

```diff
@@ -0,0 +1,43 @@
+<h4 class="exercise-title">
+    {{ exercise.title }}
+    <span
+        >[{{ exercise.maxPoints }} {{ 'artemisApp.examParticipation.points' | artemisTranslate }}]
+        <jhi-included-in-score-badge
+            *ngIf="exercise.includedInOverallScore !== IncludedInOverallScore.INCLUDED_COMPLETELY"
+            [includedInOverallScore]="exercise.includedInOverallScore"
+        ></jhi-included-in-score-badge
+    ></span>
+</h4>
+<jhi-commits-info
+    class="mb-2"
+    [currentSubmissionHash]="currentSubmission?.commitHash"
+    [previousSubmissionHash]="previousSubmission?.commitHash"
+    [participationId]="studentParticipation?.id"
+    [submissions]="submissions"
+    [exerciseProjectKey]="exercise?.projectKey"
+/>
+<div *ngIf="exercise?.gitDiffReport">
+    <dt>
+        <span jhiTranslate="artemisApp.programmingExercise.diffReport.lineStatBetweenSubmissionsLabel">Lines added/removed between current submission and previous submission</span>
+    </dt>
+    <dd>
+        <div class="fw-bold">
+            <jhi-git-diff-line-stat
+                [addedLineCount]="addedLineCount"
+                [removedLineCount]="removedLineCount"
+                ngbTooltip="{{ 'artemisApp.programmingExercise.diffReport.lineStatTooltipExamTimeline' | artemisTranslate }}"
+            ></jhi-git-diff-line-stat>
+            <jhi-button
+                *ngIf="addedLineCount > 0 || removedLineCount > 0"
+                [featureToggle]="FeatureToggle.ProgrammingExercises"
+                [isLoading]="isLoadingDiffReport"
+                [btnSize]="ButtonSize.SMALL"
+                [icon]="faEye"
+                [title]="'artemisApp.programmingExercise.diffReport.button'"
+                [tooltip]="'artemisApp.programmingExercise.diffReport.tooltipBetweenSubmissions'"
+                (onClick)="showGitDiff()"
+                class="ms-2"
+            ></jhi-button>
+        </div>
+    </dd>
+</div>
```

Comment:
`programming-exercise-exam-diff.component.scss` seems to be empty, can you remove that file?